iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1590,7 +1590,7 @@
 
 "type.barrier.chain" = "Cadena";
 
-"type.barrier.city_wall" = "Muro de la ciudad";
+"type.barrier.city_wall" = "City Wall";
```

Comment:
@RedAuburn can you please debug why your script always overwrites these strings?

Reply:
argh sorry, it's strange, sometimes it does, sometimes it doesnt. will double check in future